launcher/common/eagle3/offline_training.sh

```
@@ -0,0 +1,40 @@
+
+pip install -r modules/Model-Optimizer/examples/speculative_decoding/requirements.txt
+pip install huggingface-hub>=1.2.1
+export PATH=$PATH:/workspace/.local/bin
+
+###################################################################################################
+
+trap 'error_handler $0 $LINENO' ERR # ERROR HANDLER
+
+bash modules/Model-Optimizer/examples/speculative_decoding/launch_train.sh \
+    --model ${HF_MODEL_CKPT} \
+    ${@}
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)